hw/ip/otbn/dv/otbnsim/sim/ext_regs.py

```diff
@@ -208,7 +208,10 @@ def poison(self) -> None:
         self._client.poison()
 
     def forget(self) -> None:
+        # Clear any pending request in the RND EDN client
+        # Also clear the request flag
         self._client.forget()
+        self.write(0, True)
```

Comment:
Nit: Suggest to move the second comment directly above the corresponding code
```suggestion
        self._client.forget()
        # Also clear the request flag
        self.write(0, True)
```